src/main/java/uk/gov/hmcts/reform/slc/services/ReportParser.java

```diff
@@ -27,8 +28,8 @@
         try (CSVParser parser = parserFor(report)) {
 
             return stream(parser.spliterator(), false)
-                .map(row -> toPrintStatus(row))
```

Comment:
I think previous form worked better - made things clearer. It would be better to suppress this warning.